webui/react/src/stores/auth.ts

```diff
@@ -24,7 +24,7 @@ const internalAuth = new WritableObservable<Loadable<Auth>>(NotLoaded);
 const internalAuthChecked = new WritableObservable(false);
 
 export const auth = internalAuth.readOnly();
-export const authChecked = internalAuth.readOnly();
+export const authChecked = internalAuthChecked.readOnly();
```

Comment:
nit: its totally up to you, but renaming `isInternalAuthChecked` might be helpful to avoid this mistake.
`internalAuth` and `internalAuthChecked` are very similar 🙄

Reply:
good idea -- let's do it as part of the observable cleanup.